js/jquery.dvfTables.js

```diff
@@ -111,14 +111,6 @@
         return this;
       }
 
-      $('<button/>')
```

Comment:
@WilsonWu2100 we dot simply want to remove the button but instead remove it when the required configuration is missing
cc @jez500

Reply:
This issue was fixed now